frontend/apps/reader/modules/readerthumbnail.lua

```diff
@@ -250,6 +250,10 @@ end
 function ReaderThumbnail:getPageThumbnail(page, width, height, batch_id, when_generated_callback)
     self:setupCache()
     self.current_target_size_tag = string.format("w%d_h%d", width, height)
+    if self.ui.rolling and Screen.night_mode and self.ui.document.configurable.nightmode_images == 1 then
+        -- We'll get a different bb in this case: it needs it own cache hash
+        self.current_target_size_tag = self.current_target_size_tag .. "_ni"
```

Comment:
nit: I usually abbreviate to `nm`

Reply:
(`ni` meant "nightmode images", but ok :)